examples/tests/valid/expressions_binary_operators.w

```diff
@@ -6,5 +6,13 @@ let xyz = ((y * y) / (x * x)) * z;
 let xf = 1.0;
 let yf = (-20.22) * xf;
 let zf = (xf + (yf) - -0.01);
+let fxzy = 5 ^ (1^5);
+assert(fxzy == 5);
```

Comment:
According to the spec this is the "exponent" operator. I'm not sure what this means. It's probably supposed to be a **power** operator. But in either case what you added here acts like the javascript `^` operator which is a "bitwise xor" operator. So to clarify:
```
// JavaScript
7^3 = 4 // Bitwise XOR: binary 111 xor binary 011 = binary 100 = 4
// Power  - this is what I think we want with the ^ operator
7^3 = Math.pow(7,3) = 343
// Exponent - x^y - this is unclear. It may mean the exponent of x but then this is a unary operator and there's no need for y
Math.exp(7)
```
@staycoolcall911 I suggest we either delete `^` from the spec for now, or fix the spec to say it's a power operator (and not exponent).

Reply:
@yoav-steinberg to be clear, the "exponent" operator (https://github.com/winglang/wing/pull/1654) was jsified as `**`, which is function-wise equivalent to `Math.pow`. Tho the meaning of syntax `^` in the spec is a bit different from the javascript. 

I think it can be a separate PR if @staycoolcall911 thinks "exponent" the spec and the implementation should be removed. WDYT

Reply:
Good catch @yoav-steinberg.

I suggest:
1. Since this PR is not related to "exponent", I think @yoav-steinberg can approve it as is ("exponent" was actually added in a previous PR #1654 which is already merged in).
2. @flyingImer - could you please add another PR to fix the name of `exponent` to `power` in the spec? 
I'd say please also change the operator from `^` to `**`, because `^` is confusing. Thanks!

Happy to hear if there are other opinions/suggestions.

Reply:
And if this PR adds a test for power (exponen) lets make it a test that really shows this is power:
```suggestion
let fxzy = 5 ^ (2^3);
assert(fxzy == 390625);
```

Reply:
@yoav-steinberg will make that happen in the following PR

Reply:
Resolved the conversation and @yoav-steinberg approved your PR. Thank you for your patience and for following up @flyingImer 🙏